File: 0001_0100/0069/0069.py
import math
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    n, a = (int(x) for x in f.read().split())
    q.write("YES" if a * (1 - math.cos(math.pi / n)) / ( 2 * math.sin(math.pi / n)) < 1 else "NO")
    
           
        

File: 0001_0100/0069/0069.go
package main

import (
	"fmt"
	"math"
	"os"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var n, a float64
	fmt.Fscan(f, &n, &a)
	r := "NO"
	if a*(1-math.Cos(math.Pi/n))/(2*math.Sin(math.Pi/n)) < 1 {
		r = "YES"
	}
	fmt.Fprint(q, r)
}
